source/content/guides/autopilot/07-troubleshoot.md

```diff
@@ -367,6 +367,20 @@ Site is running a framework that is not supported. Supported frameworks are:
 
 </Accordion>
 
+## Composer configuration Not Supported
+
+<Accordion title="We could not run Autopilot because the site is running an unsupported framework." id="misconfigured-composer" icon="info-sign">
+
+### Diagnosis
+
+A [Wordpress Composer Managed](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) site which is not using Composer is not supported by Autopilot.
```

Comment:
@alynnferry just to confirm.. this is about wordpress-composer-managed sites that aren't using composer _at all_?  Or is it that they aren't using _Integrated Composer_ specifically?

Reply:
I believe it is wordpress-composer-managed sites that are not using Integrated Composer. @danilarff86 Can you confirm?

Reply:
Capitalize P please :) WordPress

Reply:
```suggestion
A [WordPress Composer Managed](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) site which is not using Composer is not supported by Autopilot.
```